locales/fluent/base/vi.ftl

```diff
@@ -0,0 +1,149 @@
+### Base localization / generic strings
+### Messages should only go into this file if they are widely used,
+### or are particularly important to localize.
+
+## Terms
+
+-service-name = Wikijump
+
+## Special
+
+# Shows whenever a message is still being loaded
+message-loading = Đang tải...
+
+goto-home = Về trang chủ
+
+goto-service = Về { -service-name }
+
+base-title = { $title } | { -service-name }
+
+navigated-to = Đã chuyển hướng tới { $path }
+
+## Generic
+
+about = Thông tin
+account = Tài khoản
+applications = Ứng dụng
+avatar = Ảnh đại diện
+breadcrumbs = Breadcrumb
+change = Thay đổi
+clear = xóa
+close = Đóng
+dashboard = Tổng quan
+docs = Tài liệu
+download = Tải xuống
+edit = Chỉnh sửa
+editor = Trình chỉnh sửa
+footer = Chân Trang
+general = Chung
+header = Đầu Trang
+help = Trợ giúp
+inbox = Hộp thư
+invitations = Lời mời
+license = Giấy phép
+load = Tải
+main-content = Nội Dung Chính
+messages = Tin nhắn
+navigation = Điều hướng
+notifications = Thông báo
+preview = Xem trước
+privacy = Riêng tư
+profile = Hồ sơ
+publish = Xuất bản
+reveal-sidebar = Mở Thanh Cạnh
+save = Lưu
+security = Bảo mật
+send = Gửi
+sent = Đã gửi
+settings = Cài đặt
+sidebar = Thanh cạnh
+tags = Tag
+terms = Điều khoản
+upload = Tải lên
+
+search = Tìm kiếm
+  .placeholder = Tìm...
+
+## Generic Authentication
+
+login = Đăng nhập
+  .toast = Bạn đã được đăng nhập.
+
+logout = Đăng xuất
+  .toast = Bạn đã được đăng xuất.
+
+register = Đăng ký
+  .toast = Bạn đã được đăng ký.
+
+specifier = Email hoặc Tên người dùng
+  .placeholder = Nhập email hoặc tên người dùng...
+
+username = Tên người dùng
+  .placeholder = Nhập tên người dùng...
+  .info = Bạn sẽ có thể thay đổi thông tin này sau.
+
+email = Email
+  .placeholder = Nhập địa chỉ email...
+  .info = Địa chỉ email của bạn sẽ là riêng tư.
+
+password = Mật khẩu
+  .placeholder = Nhập mật khẩu...
+
+confirm-password = Xác Nhận Mật Khẩu
+
+forgot-password = Quên Mật Khẩu
+  .question = Bạn đã quên mật khẩu?
+
+reset-password = Đặt Lại Mật Khẩu
+
+remember-me = Giữ đăng nhập
+
+create-account = Tạo Tài Khoản
+
+field-required = Mục này là bắt buộc
+
+characters-left = { $count ->
+  [1] 1 ký tự còn lại
+  *[other] { $count } ký tự còn lại
+}
```

Comment:
It seems like Vietnamese doesn't have a separate form for plurals. So you could just do this instead, right?
```
characters-left = { $count } ký tự còn lại
```

Reply:
you mean the plural form like "s" in english?

Reply:
Yes. The reason there is the `{ $count -> ... }` for English is because the translation is different if it's 1 vs many. It looks like "ký tự còn lại" is the same for either case, so the switch is not necessary.

Reply:
Vietnamese nouns only have 1 form for singular and plural meanings, so it stays the same

Reply:
Okay cool. Then you can change the translation line to `characters-left = { $count } ký tự còn lại` instead to reflect the 1 form for both meanings.

Reply:
I just changed and saved it, thanks for telling me about that